docs/user-guide/update-demo/index.md.orig

```diff
@@ -0,0 +1,107 @@
+---

```

Comment:
Was the file `docs/user-guide/update-demo/index.md.orig` committed accidentally or on purpose?